src/main/java/net/william278/velocitab/player/TabPlayer.java

```diff
@@ -192,4 +192,23 @@ public boolean equals(Object obj) {
         return obj instanceof TabPlayer other && player.getUniqueId().equals(other.player.getUniqueId());
     }
 
+    /**
+     * Represents a nametag to be displayed above a player, with prefix & suffix
+     *
+     * @param prefix The prefix
+     * @param suffix The suffix
+     */
+    @SuppressWarnings("SpellCheckingInspection")
+    public record Nametag(@Nullable String prefix, @Nullable String suffix) {
```

Comment:
I think prefix and suffix can't be null, so maybe it's better to flag them as NotNull